.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+python:
+  - 2.7
+  - 3.4
+notifications:
+  email: false
```

Comment:
Eh. I'm not too keen on superfluous emails. If I remove this, only the submitter should be spammed right?